modules/tests/support/model_based/ICS03.tla

```diff
@@ -331,7 +331,8 @@ ICS03_ConnectionOpenAck(
                         \* verification passed; update the connection state to
                         \* "Open"
                         LET updatedConnection == [connection EXCEPT
-                            !.state = "Open"
+                            !.state = "Open",
+                            !.counterpartyConnectionId = counterpartyConnectionId
```

Comment:
I tested this change with the old-code (as per @romac's suggestion) and the MBT tests do catch the missing `set_counterparty()` and fail.

Reply:
Awesome :)